core/search/templates/search/search_results.js

```diff
@@ -5,26 +5,21 @@ function collapseResults( group ) {
     if ( dataModel == openGroup ) {
         return;
     }
+    if ( dataModel == 'Staff Directory' ) {
```

Comment:
You'll want to define the variables outside the condition, and update them within the conditions. Same with the `numMore` variable.